terraform/modules/jenkins-infra-shared-data/outputs.tf

```diff
@@ -1,7 +1,7 @@
 output "admin_public_ips" {
   value = {
     dduportal           = ["89.84.210.161", "91.174.176.56"],
-    lemeurherve         = ["176.185.227.180"],
+    lemeurherve         = ["176.185.227.180", "2a01:e0a:ec8:e200:1911:1e0:5159:5fe5"],
```

Comment:
Alas, IPv4 is required (Azure doc. is not clear but adding an IPv6 as IP in the UI or `az` CLI fails):

- https://github.com/jenkins-infra/azure/blob/main/publick8s.tf#L34-L51
- https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster#authorized_ip_ranges
- https://learn.microsoft.com/en-us/azure/aks/api-server-authorized-ip-ranges?tabs=azure-cli